client/devdocs/design/index.jsx

```diff
@@ -50,14 +53,22 @@ import ExternalLink from 'components/external-link/docs/example';
 import FeatureGate from 'components/feature-example/docs/example';
 import FilePickers from 'components/file-picker/docs/example';
 import Collection from 'devdocs/design/search-collection';
+import fetchUsageStats from 'state/usage-stats/actions';
 
-export default React.createClass( {
+let DesignAssets = React.createClass( {
 	displayName: 'DesignAssets',
 
 	getInitialState() {
 		return { filter: '' };
 	},
 
+	componentWillMount() {
+		if ( config.isEnabled( 'devdocs/usage-stats' ) ) {
+			const { dispatchFetchUsageStats } = this.props;
+			dispatchFetchUsageStats();
```

Comment:
I don't think so unless we want to fetch the stats again for users that are visiting the design page while a new version of Calypso is live. But this will be quite a complicate thing to do and I am not sure if it is necessary/worth doing it. Did you have anything else in mind?